### DISCUSSION TITLE
Timberin temp

### DISCUSSION THREAD

**Discussion by lilnane** [theengs/decoder#266] [New devices/things integration]
Hello everyone!
I received as a gift a temperature sensor (Timberin) that can only be controlled with an Android application (TimberTemp)
Is there by any chance a way to detect it from jeedom? because I do not have an Android device ... 
Thank you very much in advance

**Comment by DigiH** [comment]
Hi @lilnane,

this really depends on the Timberin freely broadcasting advertisement data which contains the temperature in an encoded format, which could then be decoded with Decoder.

Can you see the device's encoded manufacturerdata or servicedata when monitoring your MQTT broker output with MQTT Explorer, or if you monitor your bluetooth device environment with the nRFConnect app ?

**Reply by lilnane** [reply]
![timb2](https://user-images.githubusercontent.com/123254829/213926658-9bf0d23c-b736-47bd-9109-153f38c4116c.jpg)
![timb1](https://user-images.githubusercontent.com/123254829/213926674-377e218f-2ff7-4768-9fc9-55260ed9037e.jpg)
Sorry for my English I'm French ....I don't have solution to find the device's encoded manufacturerdata with my eyes as you can see with the photo .I'm using jeedom and I have tried to detect it with the plugin bluetooth but I don't know which of them it is since there is too much device without name ...
many thanks for helping me

**Reply by DigiH** [reply]
Could you possible download[ MQTT Explorer](https://github.com/thomasnordquist/MQTT-Explorer/releases/tag/v0.3.5) and see if it might be easier to spot in its output from your broker, possiblywith it having a name included in its broadcast.

Or post a screenshot of the outputs where we might be able to eliminate some other known broadcasts.

**Reply by lilnane** [reply]
![jeedomhelp](https://user-images.githubusercontent.com/123254829/213928636-43c5721e-e1a9-4da2-8fb2-7a8a6dbba51f.jpg)
i am a real beginner in all of this so I put here this screenshot during this time I will learn what is mqtt.....so sorry !

**Reply by DigiH** [reply]
No need to apologise :)

I personally have no knowledge of Jeedom, but it only seems to display the different bluetooth devices with very little information. 

MQTT Explorer really would be the ideal application, with it showing additional broadcast data, to use for seeing if there is a recognisable device which could be the Timberin temperature sensor and if and how it can be decoded.

Do you know what MQTT broker is being used with Jeedom?

Also pinging @BadWolf42 and @1technophile, as they are more familiar with Jeedom and its MQTT side - and speak the lingo.

**Reply by 1technophile** [reply]
Salut,

Pour avoir des information detaillees sur le capteur, il faudrait que tu ailles dans l'onglet "Temps reel" du plugin jMQTT, voir ci dessous:
https://domochip.github.io/jMQTT/fr_FR/#mode-temps-r%C3%A9el
Et que tu filtres sur ce capteur en identifiant son nom par exemple

Une presentation de MQTT en francais: https://www.youtube.com/watch?v=7skzc4axLKM

**Comment by lilnane** [comment]
it's a pleasure to be welcomed like this thank you so much...I will search about MQTT broker with jeedom and come back here to give back some news !
